canonical/dotrun
# Add network_mode host for linux

## Done

- Use the `network_mode=host` when the OS is Linux: this will allow projects to communicate with other servers running on localhost such as a database

This option only will be applied on Linux because Docker only supports host mode on Linux.

## QA

- Install Poetry: https://python-poetry.org/docs/#installation
- Install deps: `poetry install`
- Go to a project, e.g. ubuntu.com and run dotrun:
```bash
poetry run -C /path/to/dotrun dotrun
```
- Make sure that the project runs as intended

## Issues

Fixes #101
